src/authentication/requestPasswordReset.ts

```diff
@@ -1,10 +1,21 @@
 import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
 import { logErrorReturnLogId } from "@/logger/logger";
+import { AuthError } from "@supabase/gotrue-js";
 
 interface RequestPasswordResetResponse {
     errorMessage: string | null;
 }
 
+function formatErrorMessage(error: AuthError): string {
+    const helpEmailAddress = "help@foodbankapp.org";
```

Comment:
let's make a file helpEmailAddress.ts in app/common and move this there

Maybe one day someone will add some more email addresses or other non sensitive useful website data to that file and rename it something more general. But for now it will be okay to just call it that! :)

Reply:
Sounds like a good plan! There isn't an app/common at the moment. Did you mean src/common? Or should I create app/common?

Reply:
sorry I meant src/common 
*facepalm*